### ISSUE TITLE
improve pull requests and commits msg for non SAP employee

### ISSUE THREAD

**Issue by johgoe** [kyma-project/kyma#11836]
We try to watch changes of kyma project to be prepared for upcomping changes.

To see the reason of a #change we check the issue or the pull requests. We see more often just a link to a sap internal platform like 
https://github.tools.sap/kyma/backlog/issues/1717
It would be great to copy the most important details to the pull request to make the informations available for non SAP employees.

Thanks in advance

**Comment by pbochynski** [comment]
Yes. You are right. Usually, we create issues in open source project for internal requests, but we need a better check. I will talk with the team to put more effort into making all issues transparent.
